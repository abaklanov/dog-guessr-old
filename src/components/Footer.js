import React from "react";
import { Button } from "react-bootstrap";

class Footer extends React.Component {
  render() {
    return (
      <div className="row">
        <Button variant="secondary" id="skip">
          Skip
        </Button>
      </div>
    );
  }
}

export default Footer;
