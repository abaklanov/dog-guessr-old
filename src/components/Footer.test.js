import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";
import { Button } from "react-bootstrap";

const wrapper = shallow(<Footer />);

it("renders correctly", () => {
  expect(wrapper).toMatchSnapshot();
});

it("displays Skip button", () => {
  const button = wrapper.find(Button).find("#skip");
  expect(button).toHaveLength(1);
});
