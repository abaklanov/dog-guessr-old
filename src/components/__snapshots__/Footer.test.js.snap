// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <ForwardRef(Bootstrap(Button))
    id="skip"
    variant="secondary"
  >
    Skip
  </ForwardRef(Bootstrap(Button))>
</div>
`;
